### PR TITLE
Navbar fixed and responsive nabber added 

### PR DESCRIPTION
**Pull request by seymourjr5**
Navbar fixed and responsive nabber added 
Screenshot 2022-07-25 at 15 26 27
Screenshot 2022-07-25 at 15 26 36
**Small Screens**
Screenshot 2022-07-25 at 15 27 06
Screenshot 2022-07-25 at 15 27 18

### CODE REVIEW

File: globle.css [hunk 1]
```diff
@@ -75,48 +68,102 @@ body::-webkit-scrollbar-thumb {
...
-  margin-bottom: 1rem;
-  text-align: center;
-  color: var(--text-secondary);
-  background: var(--bg-secondary);
-  font-size: 1.5rem;
-  letter-spacing: 0.3ch;
-  width: 100%;
+/* Cool nav bar vv  */
+
+@media only screen and (max-width: 768px){
```
Comment (ethan-master-coding) [inline globle.css:107]:
```suggestion
@media only screen and (max-width: 580px){
```

File: Contact.html [hunk 2]
```diff
@@ -5,35 +5,34 @@
     <link href="./globle.css" rel="stylesheet" type="text/css"/>
     <link href="./contact.css" rel="stylesheet" type="text/css" />
     <meta name="description" content="This is my own personl website.">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, shrink-to-fit=no">
+    <meta name="description" content="Project">
```
Comment (ethan-master-coding) [inline Contact.html:10]:
```suggestion
```

Already have a description on line 7

File: index.html [hunk 3]
```diff
@@ -3,37 +3,37 @@
   <head>
     <title>My work</title>
     <meta name="description" content="This is my own personl website.">
-    <script src="https://kit.fontawesome.com/7dc0a11abb.js" crossorigin="anonymous"></script>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, shrink-to-fit=no">
+    <meta name="description" content="Project">
```
Comment (ethan-master-coding) [inline index.html:8]:
```suggestion
```

Already have a description on line 5

**Review by ethan-master-coding — CHANGES_REQUESTED** [review]
I like the changes. On desktop when the nav bar is collapsed I would prefer that the icons still display like it is currently.

**Comment by seymourjr5** [comment]
I have changed the nav bar on desktop as you suggested 
Screenshot 2022-07-26 at 10 18 00
Screenshot 2022-07-26 at 10 18 08
**on hover**
Screenshot 2022-07-26 at 10 18 24

File: Projects.html [hunk 4]
```diff
@@ -10,31 +13,25 @@
...
-            <i class="fas fa-id-card-alt"></i>
-            <span class="link-text">Contact</span>
-          </a>
-        </li>
-      </ul>
-    </nav>
 
+    <div class="navbar">
+      <div id="nav-items">
+        <div class="nav-item"><a href="./index.html"><i class="fas fa-home links" id="nav-logo"></i> <span class="nav-text">Home</span></a></div>
```
Comment (ethan-master-coding) [inline Projects.html:19]:
```suggestion
        <div class="nav-item"><a href="https://ethan-master-coding.github.io/My-work/"><i class="fas fa-home links" id="nav-logo"></i> <span class="nav-text">Home</span></a></div>

```

File: Projects.html [hunk 5]
```diff
@@ -10,31 +13,25 @@
...
+        <div class="nav-item"><a href="./Projects.html"><i class="fas fa-project-diagram links" id="nav-logo"></i> <span class="nav-text">Projects</span></a></div>
```
Comment (ethan-master-coding) [inline Projects.html:20]:
```suggestion
        <div class="nav-item"><a href="https://ethan-master-coding.github.io/My-work/Projects"><i class="fas fa-project-diagram links" id="nav-logo"></i> <span class="nav-text">Projects</span></a></div>

```

File: Projects.html [hunk 6]
```diff
@@ -10,31 +13,25 @@
...
+        <div class="nav-item"><a href="./Contact.html"><i class="fas fa-id-card-alt links"></i> <span class="nav-text">Contact</span></a></div>
```
Comment (ethan-master-coding) [inline Projects.html:21]:
```suggestion
        <div class="nav-item"><a href="https://ethan-master-coding.github.io/My-work/Contact"><i class="fas fa-id-card-alt links"></i> <span class="nav-text">Contact</span></a></div>

```

File: Projects.html [hunk 7]
```diff
@@ -10,31 +13,25 @@
...
+    </div>
+    </div>
+
+    <!-- Navbar for small screens (max-width: 768px) -->
+    <div id="mySidenav" class="sidenav">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+      <a href="./index.html"><i class="fas fa-home" id="nav-logo"></i> Home</a>
```
Comment (ethan-master-coding) [inline Projects.html:28]:
```suggestion
      <a href="https://ethan-master-coding.github.io/My-work/"><i class="fas fa-home" id="nav-logo"></i> Home</a>
```

File: Projects.html [hunk 8]
```diff
@@ -10,31 +13,25 @@
...
+      <a href="./Projects.html"><i class="fas fa-project-diagram" id="nav-logo"></i> Projects</a>
```
Comment (ethan-master-coding) [inline Projects.html:29]:
```suggestion
      <a href="https://ethan-master-coding.github.io/My-work/Projects"><i class="fas fa-project-diagram" id="nav-logo"></i> Projects</a>
```

File: Projects.html [hunk 9]
```diff
@@ -10,31 +13,25 @@
...
+      <a href="./Contact.html"><i class="fas fa-id-card-alt"></i> Contact</a>
```
Comment (ethan-master-coding) [inline Projects.html:30]:
```suggestion
      <a href="https://ethan-master-coding.github.io/My-work/Contact"><i class="fas fa-id-card-alt"></i> Contact</a>
```

File: index.html [hunk 10]
```diff
@@ -3,37 +3,36 @@
...
-        </li>
-      </ul>
-    </nav>
-    <main>
+
+    <div class="navbar">
+      <div id="nav-items">
+        <div class="nav-item"><a href="./index.html"><i class="fas fa-home links" id="nav-logo"></i> <span class="nav-text">Home</span></a></div>
+        <div class="nav-item"><a href="./Projects.html"><i class="fas fa-project-diagram links" id="nav-logo"></i> <span class="nav-text">Projects</span></a></div>
+        <div class="nav-item"><a href="./Contact.html"><i class="fas fa-id-card-alt links"></i> <span class="nav-text">Contact</span></a></div>
```
Comment (ethan-master-coding) [inline index.html:22]:
```suggestion
        <div class="nav-item"><a href="https://ethan-master-coding.github.io/My-work/Contact"><i class="fas fa-id-card-alt links"></i> <span class="nav-text">Contact</span></a></div>
```

File: index.html [hunk 11]
```diff
@@ -3,37 +3,36 @@
...
+    </div>
+    </div>
+
+    <!-- Navbar for small screens (max-width: 768px)-->
+    <div id="mySidenav" class="sidenav">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+      <a href="./index.html"><i class="fas fa-home" id="nav-logo"></i> Home</a>
```
Comment (ethan-master-coding) [inline index.html:29]:
```suggestion
      <a href="https://ethan-master-coding.github.io/My-work/"><i class="fas fa-home" id="nav-logo"></i> Home</a>
```

File: index.html [hunk 12]
```diff
@@ -3,37 +3,36 @@
...
+      <a href="./Projects.html"><i class="fas fa-project-diagram" id="nav-logo"></i> Projects</a>
```
Comment (ethan-master-coding) [inline index.html:30]:
```suggestion
      <a href="https://ethan-master-coding.github.io/My-work/Projects"><i class="fas fa-project-diagram" id="nav-logo"></i> Projects</a>
```

File: index.html [hunk 13]
```diff
@@ -3,37 +3,36 @@
...
+      <a href="./Contact.html"><i class="fas fa-id-card-alt"></i> Contact</a>
```
Comment (ethan-master-coding) [inline index.html:31]:
```suggestion
      <a href="https://ethan-master-coding.github.io/My-work/Contact"><i class="fas fa-id-card-alt"></i> Contact</a>
```

**Comment by ethan-master-coding** [comment]
Got a few more suggestions before we merge. Mostly just change the links from relative ones to absolute ones to keep it consistent.

**Comment by seymourjr5** [comment]
Should be ready to merge, I double checked everything, should work perfectly.

**Comment by ethan-master-coding** [comment]
@seymourjr5 before we merge I want to check the changed locally to make sure nothing broke. I will do that later.

**Review by ethan-master-coding — APPROVED** [review]
Just checked and we did not brake anything.

**Pull request merged**